src/components/Layout/utils/styles.ts

```diff
@@ -0,0 +1,12 @@
+import cn from '@ably/ui/core/utils/cn';
```

Comment:
Same as was in `Header`, just moved out so it could be used in the assets added here